app/views/TaxFreeAmountView.scala.html

```diff
@@ -75,36 +75,42 @@ <h2 class="govuk-heading-m">@Messages("tai.taxFreeAmount.detailsection.heading")
                     @taxFreeAmountSummary(viewModel.taxFreeAmountSummary.summaryItems)
 
             </div>
-
-                    <h2 class = "govuk-heading-m no-print hide-for-print">@Messages("tai.incomeTaxSummary.addMissingIncome.section.heading")</h2>
-
-                    <ul class="govuk-list govuk-list--bullet govuk-list--spaced hide-for-print">
-                        <li>
-                            @link(
-                                id=Some("addMissingAddition"),
-                                url=appConfig.taxFreeAllowanceLinkUrl,
-                                copy=Messages("tai.taxFreeAmount.missing.section.addition"))
-                        </li>
-                        <li>
-                            @link(
-                                id=Some("addMissingDeduction"),
-                                url=appConfig.companyBenefitsLinkUrl,
-                                copy=Messages("add.missing.company.benefit"))
-                        </li>
-                        <li>
-                            @link(
-                                id=Some("addMissingInvestment"),
-                                url=appConfig.investmentIncomeLinkUrl,
-                                copy=Messages("tai.taxFreeAmount.missing.section.investment"))
-                        </li>
-                        <li>
-                            @link(
-                                id=Some("addMissingIncome"),
-                                url=appConfig.otherIncomeLinkUrl,
-                                copy=Messages("tai.taxFreeAmount.missing.section.income"))
-                        </li>
-                    </ul>
-
+            <div class="govuk-!-margin-top-2">
+                <h2 class = "govuk-heading-m no-print hide-for-print">@Messages("tai.taxFreeAmount.missing.additionsOrDeductions.heading")</h2>
+                <p class="govuk-body govuk-body-m">@Messages("tai.taxFreeAmount.missing.additionsOrDeductions.paragraph")</p>
```

Comment:
it's missing the css classes: no-print hide-for-print.

See Jira. I am not sure this entire section should be hidden when printing.